### ISSUE TITLE
exclude does not work for axe

### ISSUE THREAD

**Issue by IConstructor** [angular/protractor-accessibility-plugin#33]
Hello guys. I am using protractor-accessibility-plugin and I want to exclude from testing the application's header (I know class, id etc), however, my "exclude" in configuration does not work. At the same time, runOnly restriction works from my config. What am I doing wrong?
plugins: [{
axe: {
exclude: [
['#divTopbar']
],
...
